js/publisher.js

```diff
@@ -24,24 +26,19 @@ class JS extends Publisher {
     this.loadStubs(['publisher'])
   }
 
-  beforeEach (mockery) {
-    this.ledger.setSynopsis(null)
+  runBeforeEach (mockery) {
+    this.beforeEach(mockery)
+    this.changeSetting('PAYMENTS_ALLOW_NON_VERIFIED', false)
```

Comment:
Just curious, why are we not allowing non-verified here?

Reply:
This was an oversight, removing, thanks for catching.

Reply:
fixed